examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -26,19 +93,48 @@
 def main(cfg: DictConfig) -> None:
     logging.info(f'Config: {cfg.pretty()}')
     trainer = pl.Trainer(**cfg.trainer)
-    exp_manager(trainer, cfg.get("exp_manager", None))
+    exp_dir = exp_manager(trainer, cfg.get("exp_manager", None))
 
-    question_answering_model = QAModel(cfg.model, trainer=trainer)
-    trainer.fit(question_answering_model)
+    do_training = True
+    if not cfg.pretrained_model:
+        logging.info(f'Config: {OmegaConf.to_yaml(cfg)}')
+        model = QAModel(cfg.model, trainer=trainer)
+    else:
+        logging.info(f'Loading pretrained model {cfg.pretrained_model}')
+        # TODO: Remove strict, when lightning has persistent parameter support for add_state()
+        model = QAModel.from_pretrained(cfg.pretrained_model, strict=False)
+        if do_training:
+            model.setup_training_data(train_data_config=cfg.model.train_ds)
+            model.setup_validation_data(val_data_config=cfg.model.validation_ds)
+
+    if do_training:
```

Comment:
Why do you have this do_training?

Reply:
to showcase difference usages. If you do do_training=False, then it will not train the model and continue with inference/evaluation. I followed the token_classification.py for this.